pages/apply.js

```diff
@@ -1,81 +1,18 @@
-import styled from 'styled-components'
-
-import { Heading, Form, Page } from 'components'
-import { MOBILE_SIZE, TABLET_LARGE_SIZE } from 'styles/constants'
-
-const HeroSection = styled.section`
```

Comment:
I thought we were using styled components for consistency

Reply:
I followed the structure of the home and about pages. We should make a standard, and I can make the changes. This can be a fix before launch, however?